src/main/java/com/google/sps/data/ProfileBuilder.java

```diff
@@ -13,42 +13,43 @@
 // limitations under the License.
 
 package com.google.sps.data;
+import java.util.ArrayList;
 
 /** class to build profile */
 public class ProfileBuilder {
   // required
-  private long id;
-  private String userName;
+  public String id;
+  public String userName;
 
   // optional
-  private boolean vegetarian;
-  private boolean vegan;
-  private boolean glutenFree;
-  private boolean dairyFree;
-  private String[] allergies;
+  public boolean vegetarian;
+  public boolean vegan;
```

Comment:
Just curious why you decided to make private vs. public

Reply:
I needed to make the ProfileBuilder variables public in order to access those variables when calling them in Profile's constructor. But I think I will create get methods in ProfileBuilder later on instead of using public variables.